packages/blade-old/src/atoms/TextInput/TextInput.native.js

```diff
@@ -405,7 +385,7 @@ const TextInput = React.forwardRef(
                                   hasLeftIcon={hasLeftIcon}
                                   maxLength={maxLength}
                                   onLayout={onTextInputLayout}
-                                  value={input}
+                                  value={value}
```

Comment:
_TextInput is a controlled component, which means the native value will be forced to match this value prop if provided_
This is a breaking change and value would need to be provided for every TextInput in the app.

Reply:
Already did a sanity for all the input once merge will give QA for sanity as well.

In my knowledge in design system we should not control value internally. This should be upto the client. Please do correct me if I am wrong

Reply:
@AditiTiwariRazorpay we need to cross-check with PG mobile devs too, this is a breaking change for them too and we shall provide a migration path for existing consumers before merging this PR

@shubhnik @shridhar52war